packages/tailwindcss-language-server/src/project-locator.ts

```diff
@@ -589,6 +611,29 @@ class FileEntry {
     await Promise.all(this.deps.map((entry) => entry.resolveRealpaths()))
   }
 
+  async resolveSourceDirectives() {
+    if (this.sources.length > 0) {
+      return
+    }
+    // Note: This should eventually use the DesignSystem to extract the same
+    // sources also discovered by tailwind. Since we don't have everything yet
+    // to initialize the design system though, we set up a simple postcss at
+    // rule exporter instead for now.
+    await postcss([
+      {
+        postcssPlugin: 'extract-at-rules',
+        AtRule: {
+          source: ({ params }: AtRule) => {
+            if (params[0] !== '"' && params[0] !== "'") {
+              return
+            }
+            this.sources.push(params.slice(1, -1))
+          },
+        },
+      },
+    ]).process(this.content, { from: this.realpath })
```

Comment:
This makes me realize we're going to parse stuff with PostCSS probably a lot more than is necessary. There's no reason to change it right now but it's definitely something we should address in the future.